x-pack/test/security_solution_endpoint/apps/endpoint/resolver.ts

```diff
@@ -194,7 +194,8 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await (await testSubjects.find('resolver:graph-controls:zoom-in')).click();
       });
 
-      it('Check Related Events for event.file Node', async () => {
+      // SKIP: https://github.com/elastic/kibana/issues/83446
```

Comment:
@IgorGuz2000 a heads up that we will be skipping these tests to unblock this and downstream work blocked on enabling APM here.

Reply:
Spoke with @IgorGuz2000, this should be resolved so I am reverting this skip